src/base/static/state/ducks/datasets-config.js

```diff
@@ -29,38 +29,43 @@ export const datasetsConfigPropType = PropTypes.arrayOf(
   }),
 ).isRequired;
 
+// TODO: refactor these to use Datasets, not DatasetConfigs, and move into a
+// "permissions" util.
 export const hasAnonAbilitiesInAnyDataset = ({
```

Comment:
Neither of these methods are querying that much state, so I don't think we need to have them in our ducks.

I think we should eventually move towards a "permissions" utility. They can accept a Dataset instance, or User instance as needed, without having to split the methods across our Dataset and User ducks.